flang/include/flang/Optimizer/Builder/IntrinsicCall.h

```diff
@@ -25,6 +25,7 @@
 namespace fir {
 
 class StatementContext;
+struct IntrinsicHandlerEntry;
 
 // TODO: Error handling interface ?
 // TODO: Implementation is incomplete. Many intrinsics to tbd.
```

Comment:
You might double check if these comments are still useful and accurate.

Reply:
(The comment about `genIntrinsicCall` looks odd.)

Reply:
Thanks Val, these TODO are not very helpful/accurate here anymore. I removed them. I also rephrased the `genIntrinsicCall` comment that pointed to another version "above" that does not exists anymore.